Detect devices for brands: Irbis, Globex, Megafon, MTC, Sony, Haier, Ginzzu, Atom, Highscreen, Lenovo, Senseit, Mann, Umax

sanchezzzhak opened the pull request.

sanchezzzhak committed:
feat(devices): added detect Irbis TZ960;
fix HTC detect Wildfire E
detect Fly Era Energy 2 (IQ4401)
detect Wiko View 3 Pro (W-P611-EEA)
detect Assistant AP-110N

sanchezzzhak committed:
feat(device) detect Globex X8 (GU8012C)

sanchezzzhak committed:
feat(device) detect Megafon Login 3 (MFLogin3T)

sanchezzzhak committed:
feat(device) detect Sony Xperia Tablet Z4 (SGP712)

sanchezzzhak committed:
feat(device) detect brand Haier models: Titan T1, Titan T3

sanchezzzhak committed:
Merge remote-tracking branch 'upstream/master' into pr_device_29

sanchezzzhak committed:
feat(device) detect Lenovo Vibe Z2

sanchezzzhak committed:
feat(device) detect brand: Atom - 108AM (smart tv/mediaplayer)

sanchezzzhak committed:
feat(device) detect Highscreen Power Five Pro

sanchezzzhak committed:
feat(device) detect Highscreen Power Rage Evo

sanchezzzhak committed:
feat(device) detect Highscreen Power Four

sanchezzzhak committed:
feat(device) detect MTC Smart Run 4G

sanchezzzhak committed:
fix(device) detect Senseit T300

sanchezzzhak committed:
feat(device) detect Ginzzu ST6120

sanchezzzhak committed:
Merge remote-tracking branch 'upstream/master' into pr_device_29

sanchezzzhak committed:
feat(device) detect brand Qumo models: Vega 782, Quest 353, Quest 570
Quest 600, Vega 783, Quest 402, Yooda, Sirius 101-4G 8GB Black, Vega 8002
Altair 701 8GB Black, Quest 458, Altair 7004, Quest 476, Quest 354

sgiehl commented on regexes/device/mobiles.yml:
```diff
@@ -13025,12 +13039,45 @@ BDF:
+  regex: 'QUMO[ _]?|(Vega[ _]?78[23])[;)/ ]'
+  device: 'tablet'
+  models:
+    - regex: '(?:QUMO[ _]?)Vega[ _]?(78[23]|8002)[;)/ ]'
```
```suggestion
    - regex: '(?:QUMO[ _]?)?Vega[ _]?(78[23]|8002)[;)/ ]'
```

sgiehl commented on regexes/device/mobiles.yml:
```diff
@@ -118,7 +118,7 @@ HTC:
     - regex: 'HTC;(?: )?([a-z0-9 ]+)'
       model: '$1'
-    - regex: '(Desire|Sensation|Evo ?3D|IncredibleS|Wildfire|Butterfly)[ _]?([^;/]+) Build'
+    - regex: '(Desire|Sensation|Evo ?3D|IncredibleS|Wildfire|Butterfly)[ _]?([^;/]+)(?: Build|\))'
```
```suggestion
    - regex: '(Desire|Sensation|Evo ?3D|IncredibleS|Wildfire|Butterfly)[ _]?([^;/)]+)(?: Build|\))'
```

Guess the `)` would be needed, as it might match too much otherwise. wouldn't it?

sgiehl commented on regexes/device/mobiles.yml:
```diff
@@ -56,7 +56,7 @@ SFR:
 # HTC
 HTC:
-  regex: 'HTC|Sprint (?:APA|ATP)|ADR(?!910L)[a-z0-9]+|NexusHD2|Amaze[ _]4G[);/ ]|(Desire|Sensation|Evo ?3D|IncredibleS|Wildfire|Butterfly)[ _]?([^;/]+) Build|(Amaze[ _]4G|One ?[XELSV\+]+)[);/ ]|SPV E6[05]0|One M8|X525a|PG86100|PC36100|XV6975|PJ83100[);/ ]|2PYB2|2PZC5|0PJA10|0PJA2'
+  regex: 'HTC|Sprint (?:APA|ATP)|ADR(?!910L)[a-z0-9]+|NexusHD2|Amaze[ _]4G[);/ ]|(Desire|Sensation|Evo ?3D|IncredibleS|Wildfire|Butterfly)[ _]?([^;/]+)(?: Build|\))|(Amaze[ _]4G|One ?[XELSV\+]+)[);/ ]|SPV E6[05]0|One M8|X525a|PG86100|PC36100|XV6975|PJ83100[);/ ]|2PYB2|2PZC5|0PJA10|0PJA2'
```
```suggestion
  regex: 'HTC|Sprint (?:APA|ATP)|ADR(?!910L)[a-z0-9]+|NexusHD2|Amaze[ _]4G[);/ ]|(Desire|Sensation|Evo ?3D|IncredibleS|Wildfire|Butterfly)[ _]?([^;/)]+)(?: Build|\))|(Amaze[ _]4G|One ?[XELSV\+]+)[);/ ]|SPV E6[05]0|One M8|X525a|PG86100|PC36100|XV6975|PJ83100[);/ ]|2PYB2|2PZC5|0PJA10|0PJA2'
```

sgiehl commented on regexes/device/mobiles.yml:
```diff
@@ -13025,12 +13039,45 @@ BDF:
+
+# Globex
+Globex:
+  regex: '(GU8012C)[;)/ ]'
```
```suggestion
  regex: 'GU8012C[;)/ ]'
```

there is currently no need for those brakets

sgiehl reviewed (changes requested):
Left some suggestions you can simply apply

sanchezzzhak committed:
fix(deivce) regular Globex GU8012C

Co-Authored-By: Stefan Giehl <stefan@matomo.org>

sanchezzzhak committed:
fix(deivce) HTC regular

Co-Authored-By: Stefan Giehl <stefan@matomo.org>

sanchezzzhak committed:
Update regexes/device/mobiles.yml

Co-Authored-By: Stefan Giehl <stefan@matomo.org>

sanchezzzhak committed:
fix(device) QUMO Vega

Co-Authored-By: Stefan Giehl <stefan@matomo.org>

sanchezzzhak committed:
feat(devices): added detect Huawei Ascend G7 (G7-L01)

sanchezzzhak committed:
fix(device): replace old name Honor Tablet Note (T1-A21w) to new MediaPad T1 10

sanchezzzhak committed:
fix(device): replace old name Lenovo Yoga Tablet 2-1050F to new Yoga Tablet 2 10.1" and add new variant test;
fix(device): replace old name Lenovo YOGA Tablet 2-830L to new Yoga Tablet 2 8.0" and add new variant test (YOGA Tablet 2-830L, YOGA Tablet 2-830LC, YOGA Tablet 2-830F);
feat(device) detect Lenovo Yoga Tablet 2 Pro 13.3" (YOGA Tablet 2 Pro-1380F)

sanchezzzhak committed:
feat(device) detect Ginzzu (GT-X831, GT-7810)

sanchezzzhak committed:
feat(device) detect new brand Umax VisionBook 7Qi 3G

sanchezzzhak committed:
fix(device) Ginzzu tablet test (GT-7810, GT-X831, GT-7105)

sanchezzzhak committed:
feat(device) detect new brand Mann (ZUG 5S Q, ZUG 5S, ZUG 3)

sanchezzzhak committed:
feat(device) detect Umax (VisionBook 10Qi 3G, VisionBook P55 LTE Pro)
